articles/nodejs/npm.md

```diff
@@ -1,3 +1,70 @@
-# NPM and package management
+# npm and package management
 
-TODO
+npm allows you to install and manage packages for use in your Node.js applications. If you're unfamiliar with
+npm, and want to learn more, go to the [npm Documentation](https://docs.npmjs.com/).
+
+There are several ways you can work with npm in Node.js tools for Visual Studio:
+1. [npm install window](#npmInstallWindow)
+1. [Managing installed packages in the Solution Explorer](#solutionExplorer)
+1. [`.npm` in the Node.js Interactive Window](#interactive)
+
+These all work together and synchronize with the project system, and the `package.json` in the project.
+
+## <a name="npmInstallWindow"></a>npm install window
+The easiest way to install npm packages is through npm Install Window. To access that right-click on the npm node in the project and select Install New npm Packages.
+
+![Install new npm package from solution explorer](../../images/node/solution-explorer-install-package.png)
+
+In the window you can search for a package, specify options, and install. 
+
+![Search npm package](../../images/node/search-package.png)
+
+* **Dependency type** - allows you to chose between Standard, Development, and Optional. 
+Standard indicates the package is a runtime dependency, where 
+Development indicates it's only required during development.
+* **Add to package.json** - Deprecated
```

Comment:
What does it mean for this to be deprecated?

Reply:
it doesn't do anything

Reply:
Does that mean we can remove it?

Reply:
Yes, it's on my list of enhancements to this window.